### PR TITLE
EA doc enhancements

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Contributes to https://github.com/elastic/security-docs/issues/4913.

Previews:
* [Entity risk scoring](https://security-docs_bk_5038.docs-preview.app.elstc.co/guide/en/security/master/entity-risk-scoring.html):
  * Changed the risk scoring inputs bullets into a table.
  * Restructured paragraphs in the **How is risk score calculated?** section into a numbered list.
* [Asset criticality](https://security-docs_bk_5038.docs-preview.app.elstc.co/guide/en/security/master/asset-criticality.html): Updated **prioritize alerts associated with business-critical entities** link.
* [View and analyze risk score data](https://security-docs_bk_5038.docs-preview.app.elstc.co/guide/en/security/master/analyze-risk-score-data.html): Consolidated **Triage alerts associated with high-risk entities** and **Triage alerts associated with business-critical entities** into a single section.
* [Hosts page](https://security-docs_bk_5038.docs-preview.app.elstc.co/guide/en/security/master/hosts-overview.html): Added a short description for each host details flyout section.
* [Users page](https://security-docs_bk_5038.docs-preview.app.elstc.co/guide/en/security/master/users-page.html): Added a short description for each user details flyout section.

Twin serverless PR:  https://github.com/elastic/staging-serverless-security-docs/pull/316

### CODE REVIEW

File: docs/advanced-entity-analytics/analyze-risk-score-data.asciidoc [hunk 1]
```diff
@@ -48,65 +48,50 @@ Learn more about <<customize-the-alerts-table, customizing the Alerts table>>.
...
-==== Triage alerts associated with business-critical entities
-
-To analyze alerts associated with business-critical entities, you can filter or group them by entity asset criticality.
+image::images/filter-by-asset-criticality.png[Filter alerts by asset criticality level]
 
-NOTE: If you change the entity's criticality level after an alert is generated, that alert document will include the original criticality level and will not reflect the new criticality level.
+* To group alerts by entity risk level or asset criticality level, select **Group alerts by**, then select **Custom field** and search for:
 
-* Use the drop-down filter controls to filter alerts by asset criticality level. To do this, <<drop-down-filter-controls, edit the default controls>> to filter by `user.asset.criticality` or `host.asset.criticality`:
+** `host.risk.calculated_level` or `user.risk.calculated_level` for entity risk level
```
Comment (natasha-moore-elastic) [inline docs/advanced-entity-analytics/analyze-risk-score-data.asciidoc:72]:
```suggestion
** `host.risk.calculated_level` or `user.risk.calculated_level` for entity risk level:
```

File: docs/advanced-entity-analytics/analyze-risk-score-data.asciidoc [hunk 2]
```diff
@@ -48,65 +48,50 @@ Learn more about <<customize-the-alerts-table, customizing the Alerts table>>.
...
 +
 [role="screenshot"]
-image::images/filter-by-asset-criticality.png[Filter alerts by asset criticality level]
+image::images/group-by-host-risk-level.png[Alerts grouped by host risk levels]
 
-* To group alerts by asset criticality level, select **Group alerts by**, then select **Custom field** and search for `host.asset.criticality` or `user.asset.criticality`.
+** `host.asset.criticality` or `user.asset.criticality` for asset criticality level
```
Comment (natasha-moore-elastic) [inline docs/advanced-entity-analytics/analyze-risk-score-data.asciidoc:77]:
```suggestion
** `host.asset.criticality` or `user.asset.criticality` for asset criticality level:
```

**Review by nastasha-solomon — APPROVED** [review]
Lots of great improvements here - LGTM!

**Review by nastasha-solomon — APPROVED** [review]

File: docs/advanced-entity-analytics/entity-risk-scoring.asciidoc [hunk 3]
```diff
@@ -29,10 +35,12 @@ The resulting entity risk scores are stored in the `risk-score.risk-score-<space
 [[how-is-risk-score-calculated]]
 == How is risk score calculated?
 
-The risk scoring engine runs hourly to aggregate `Open` and `Acknowledged` alerts from the last 30 days. For each entity, the engine processes up to 10,000 alerts. It groups alerts by `host.name` or `user.name`, and aggregates the individual alert risk scores (`kibana.alert.risk_score`) such that alerts with higher risk scores contribute more than alerts with lower risk scores. The resulting aggregated risk score is assigned to the **Alerts** category in the entity's <<host-risk-summary, risk summary>>.
+. The risk scoring engine runs hourly to aggregate `Open` and `Acknowledged` alerts from the last 30 days. For each entity, the engine processes up to 10,000 alerts.
```
Comment (benironside) [inline docs/advanced-entity-analytics/entity-risk-scoring.asciidoc:38]: This is an interesting detail, I didn't know this.

File: docs/getting-started/users-page.asciidoc [hunk 4]
```diff
@@ -62,9 +62,9 @@ In addition to the user details page, relevant user information is also availabl
 
 The user details flyout includes the following sections:
 
-* <<user-risk-summary, User risk summary>>
-* <<user-asset-criticality-section, Asset Criticality>>
-* <<user-observed-data, Observed data>>
+* <<user-risk-summary, User risk summary>>, which displays user risk data and inputs.
```
Comment (benironside) [inline docs/getting-started/users-page.asciidoc:65]: I like this too, it's like a table of contents with short descriptions.

**Pull request merged**